src/Analyzers/CSharp/Tests/UseCollectionInitializer/UseCollectionInitializerTests_CollectionExpression.cs

```diff
@@ -2937,7 +2938,8 @@ class C
             {
                 void M()
                 {
-                    List<int> c = [];
+                    List<int> c = 
+                    [];
```

Comment:
i don't think this is a desirable change.  i think we should only be doing your new logic when there's a comment, not just a newline.